src/validation.cpp

```diff
@@ -4179,7 +4034,7 @@ bool CChainState::ReplayBlocks(const CChainParams& params, CCoinsView* view)
 }
 
 bool ReplayBlocks(const CChainParams& params, CCoinsView* view) {
-    return g_chainstate.ReplayBlocks(params, view);
+    return ::ChainstateActive().ReplayBlocks(params, view);
```

Comment:
in commit 4d66886 refactoring: introduce ChainstateActive()

Same here (and all other changes in this commit)